src/vs/base/browser/ui/actionbar/actionbar.ts

```diff
@@ -670,7 +669,7 @@ export class ActionBar extends EventEmitter implements IActionRunner {
 
 	public dispose(): void {
 		if (this.items !== null) {
-			lifecycle.dispose(this.items);
+			this.clear();
```

Comment:
Can we not do this for stable, but only on insiders? Seems like an additional change on top that is not needed to fix the actual issue.